cdap-app-fabric/src/main/java/io/cdap/cdap/internal/capability/CapabilityConfig.java

```diff
@@ -70,6 +73,14 @@ public String getCapability() {
     return applications;
   }
 
+  /**
+   * @return {@link List} of hubs from which resources will be auto installed when this capability is enabled.
+   * Could be null.
```

Comment:
Why allow null? Shouldn't it be an empty List if there is no hub configured? also, it's better to use URL/URI as the type rather than String.

Reply:
Changed to use URL type.
I'm allowing null just to be safe - we have existing capabilities that don't have hub configured. We can fix their capability json to add empty list of hubs, but I was worried about hitting some corner case where where the json doesn't have hubs config, resulting in NPE (in particular, upgrade scenarios with accelerators configured).

Reply:
fixed to return empty list if hubs is null